config/manifests/bases/openshift-builds-operator.clusterserviceversion.yaml

```diff
@@ -26,8 +26,8 @@ metadata:
     operatorframework.io/arch.arm64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: openshift-builds-operator.v1.1.0
-  namespace: openshift-builds
+  name: openshift-builds-operator.v0.0.0
```

Comment:
We have the name for the [bundle](https://github.com/redhat-openshift-builds/operator/blob/e3f4e621eaca304c6c983a79d0f143a8e20720f5/bundle/manifests/openshift-builds-operator.clusterserviceversion.yaml#L61) set with version as `v1.1.0`. Is this change to `v0.0.0` for the operator intentional?

Reply:
This is what `operator-sdk` does in its scaffolding. The version gets updated and injected into `bundle/manifests` by running `make bundle`. We don't have automated mechanism for ensuring `make bundle` is run before code is checked in (yet).